openquake/hazardlib/valid.py

```diff
@@ -34,7 +34,8 @@
 from openquake.baselib import config, hdf5
 from openquake.hazardlib import imt, scalerel, gsim, pmf, site, tom
 from openquake.hazardlib.gsim.base import registry, gsim_aliases
-from openquake.hazardlib.calc.filters import IntegrationDistance, floatdict  # needed
+from openquake.hazardlib.calc.filters import (
+    IntegrationDistance, floatdict)  # needed
```

Comment:
@micheles do you confirm that IntegrationDistance and floatdict actually need to be imported, as the comment suggests?

Reply:
Yes